src/UserFunctions.java

```diff
@@ -70,18 +73,25 @@ public bool parseCommand(String input) {
                 break;
             default:
                 guessWord(command);
+                break;
         }
 
         return true;
     }
 
-    private void printCommands() {
+    private String getNewLineCharacter() {
         String newline = "\n";
         String osName = System.getProperty("os.name");
         // check the OS because Windows is stupid and uses two characters for a newline
         if (osName.startsWith("Windows"))
             newline = "\r\n";
 
+        return newline;
+    }
```

Comment:
I don't know that this newline function is strictly necessary (I think Windows will work with a /n), but I don't see any harm in having this.

Reply:
It may be fine on the command line, but I have already had instances where Windows didn't recognize a \n as a new line. It's hit or miss, so I figured it would be safer to check.

Reply:
Sounds good.